website/source/api/auth/approle/index.html.md

```diff
@@ -69,9 +69,15 @@ enabled while creating or updating a role.
 - `role_name` `(string: <required>)` - Name of the AppRole.
 - `bind_secret_id` `(bool: true)` - Require `secret_id` to be presented when
   logging in using this AppRole.
-- `bound_cidr_list` `(array: [])` - Comma-separated string or list of CIDR
+- `bound_cidr_list` `(array: [])` - Deprecated: Comma-separated string or list of CIDR
```

Comment:
I would just remove this field from the docs. Those who are using this will get a warning in the response. But those who are not already using it, will naturally start using the new one.

Reply:
Ah! Good idea.